_rules/SC4-1-1-unique-id.md

```diff
@@ -20,7 +20,7 @@ authors:
 
 ### Applicability
 
-Any `id` attribute specified on an HTML or SVG element.
+Any `id` attribute which is not an empty string, specified on an HTML or SVG element.
```

Comment:
> which is not an empty string

---> 

> which is not the empty string (""),

Reply:
... to align perfectly with https://github.com/auto-wcag/auto-wcag/pull/430/files